pytensor/sparse/basic.py

```diff
@@ -2268,36 +2268,49 @@ def add(x, y):
         raise NotImplementedError()
 
 
-def sub(x, y):
+def subtract(
+    x: SparseVariable | TensorVariable, y: SparseVariable | TensorVariable
+) -> SparseVariable:
     """
     Subtract two matrices, at least one of which is sparse.
 
-    This method will provide the right op according
-    to the inputs.
+    This method will provide the right op according to the inputs.
 
     Parameters
     ----------
-    x
+    x: SparseVariable or TensorVariable
         A matrix variable.
-    y
+    y: SparseVariable or TensorVariable
```

Comment:
```suggestion
    x : SparseVariable or TensorVariable
       A matrix variable.
    y : SparseVariable or TensorVariable
```

Reply:
Spaces need. Same goes below

Reply:
It's for the docs to render correctly

Reply:
Going off of this
https://numpydoc.readthedocs.io/en/latest/format.html#parameters

Reply:
or write the documentation in French